python/helper-image/launcher/launcher.go

```diff
@@ -353,6 +361,47 @@ func determinePythonMajorMinor(ctx context.Context, launcherBin string, env env)
 	return
 }
 
+// handlePydevModule applies special pydevd handling for a python module.  When a module is
+// found, we write out a python script that uses runpy to invoke the module.
+func handlePydevModule(args []string) (string, []string, error) {
+	switch {
+	case len(args) == 0:
+		return "", nil, fmt.Errorf("no python command-line specified") // shouldn't happen
+	case !strings.HasPrefix(args[0], "-"):
```

Comment:
what if it's something like `python -[some-other-flag] app.py`? is this not valid? In that case it wouldn't be recognized as a file

Reply:
if I update my entry point to this `ENTRYPOINT ["python", "-E", "app.py"]` debug suddenly breaks (it works without the `-E` flag.

Reply:
update: I may have been wrong here. running another test. will update this comment

Reply:
It does break.  That's going to be more involved — I'll do tackle it as a follow-up.

Reply:
sg.

Reply:
#76